### PR TITLE
Adding collaborators

### PR DESCRIPTION
**Pull request by mikesholiu**
Right now in my wikis controller, in the 'update' method I am adding collaborators (that are sent through a pick list in the view). When I do this for the first time (on a wiki that does not yet have any users assigned to it), it works and >>user.user_ids in the console returns the right list.

However, if it's a wiki that does already have users assigned to it (either a newly created [not seeded] wiki, or a wiki where I added collaborators already), it fails and just returns the same array that it already contained (can't add or remove collaborators).

### CODE REVIEW

**Comment by mikesholiu** [comment]
think this is mostly done now.

File: app/controllers/wikis_controller.rb [hunk 1]
```diff
@@ -28,7 +50,12 @@ def edit
 
   def update
     @wiki = Wiki.find(params[:id])
-    if @wiki.update_attributes(params.require(:wiki).permit(:title, :body))
+    @collaborators = User.where(:id => params[:collaborators])
+    @wiki.users = @collaborators 
+    if @wiki.creator? && !@wiki.users.include?(User.find(@wiki.creator))
```
Comment (chug2k) [inline app/controllers/wikis_controller.rb:55]: Do you have to do `User.find(@wiki.creator)` or could you just do `@wiki.creator`?

File: app/controllers/wikis_controller.rb [hunk 2]
```diff
@@ -28,7 +50,12 @@ def edit
...
+      @wiki.users << User.find(@wiki.creator)
```
Comment (chug2k) [inline app/controllers/wikis_controller.rb:56]: I think I would put a comment here - this code ideally should not run, right? This is a weird case.

File: app/controllers/wikis_controller.rb [hunk 3]
```diff
@@ -39,9 +66,9 @@ def update
 
   def destroy
     @wiki = Wiki.find(params[:id])
-    if @wiki.destroy
+    if @wiki.delete
```
Comment (chug2k) [inline app/controllers/wikis_controller.rb:69]: ah, why did you change this from `destroy` to `delete`?

File: app/models/user.rb [hunk 4]
```diff
@@ -1,11 +1,12 @@
 class User < ActiveRecord::Base
 
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :trackable, :validatable, :omniauthable #:confirmable
+         :recoverable, :rememberable, :trackable, :validatable, :omniauthable #confirmable
```
Comment (chug2k) [inline app/models/user.rb:4]: this will break seeds.rb, as seeds.rb has `skip_confirmation!`

**Comment by chug2k** [comment]
nothing major, just some tiny things here and there
